src/ontology/mondo-ingest.Makefile

```diff
@@ -83,7 +82,7 @@ $(COMPONENTSDIR)/ordo.owl: $(TMPDIR)/ordo_relevant_signature.txt config/properti
 		query \
 			--update ../sparql/fix_partof.ru \
 			--update ../sparql/fix_deprecated.ru \
-			--update ../sparql/fix_complex_reification.ru \
+			--update ../sparql/fix_complex_reification_ordo.ru \
```

Comment:
## SPARQL renamings: Similar issues?
@twhetzel I noticed there was another file where the namespace of the component being operated on was not in the filename. Do you mind if I do another quick PR for this? It's low priority but also quick.

`fix_make_omim_exact.ru`: This file actually corrects something to do with OMIM, but it is only run on the _DOID_ component. It's not a mistake though. It's just updating some matches in that file. For namespacing change, I'd recommend `doid_fix_make_omim_refs_exact.ru`. But if I do the source namespace to the front, that makes it consistent w/ our repo but not with the `sparql/` folder. So I could also take the time to standardize those other names in the same PR.

I could also make a follow-up issue to look for more of these, or add to a grouping issue of similar code cleanliness as I proposed in my other comment, since it's a low priority.